nixos/modules/services/web-apps/dex.nix

```diff
@@ -121,9 +121,7 @@ in
         RestrictSUIDSGID = true;
         SystemCallArchitectures = "native";
         SystemCallFilter = [ "@system-service" "~@privileged @setuid @keyring" ];
-        TemporaryFileSystem = "/:ro";
```

Comment:
When we remove this we can probably enable ProtectSystem again.

Reply:
Quickly tested on a system and seems to work.